network.py

```diff
@@ -33,11 +34,16 @@ def put(self, pkt, block=False):
 # the fields necessary for the completion of this assignment.
 class NetworkPacket:
     ## packet encoding lengths 
-    dst_addr_S_length = 5
+    dst_addr_S_length = 8
```

Comment:
why does this need to be changed?

Reply:
That var needs to be renamed. He set that up as a destination addr with 5 chars when in reality it should only need to be 1; unless I misinterpreted how his network is set up. I'm using it as a max header length and that is currently 8 to hold all information

Reply:
k, rename it and then we'll be gud